pkg/minikube/registry/drvs/kvm2/kvm2.go

```diff
@@ -87,6 +91,28 @@ func configure(mc config.MachineConfig) interface{} {
 }
 
 func status() registry.State {
+	// get libvirt id
+	virtid, err := group.LookupGid("libvirt")
```

Comment:
Is there a case that the current check fails but this one succeeds?

I ask because checking specifically for `libvirt` will not be correct in all cases. Some distros use `libvirtd`. Other configurations may use something else entirely.

Reply:
Have outlined the investigation in here https://github.com/kubernetes/minikube/issues/5617#issuecomment-554728642